CHANGES.txt

```diff
@@ -86,6 +86,15 @@
   (The culprit is boostrap commit
   `853b69f <https://github.com/twbs/bootstrap/commit/853b69f2d>`_.)
 
+- Removed deform.js:focusFirstInput()
+  Forms have an optional focus parameter ('on'|'off')
+  'on': if the form contains an input with autofocus='on', that input field 
+        will receive focus.
+  'on': if the form does not contain an input with autofocus='on', the first 
+        input field of the form will receive focus.
+  'off' disables focusing for that form.
+  Default behaviour is 'on' (unchanged).
+
```

Comment:
1. This should be moved under the section `2.0.6 (unreleased)`.
2. [Unexpected indentation](https://travis-ci.org/Pylons/deform/jobs/356946946#L595) is causing the docs build to fail. This should be:

```rst
- Removed deform.js:focusFirstInput(). Forms have an optional focus parameter
  ('on'|'off'):

    - 'on': if the form contains an input with autofocus='on', that input
      field will receive focus.
    - 'on': if the form does not contain an input with autofocus='on', the
      first input field of the form will receive focus.
    - 'off' disables focusing for that form.

  Default behaviour is 'on' (unchanged).
```

Test with https://pandoc.org/try/?text=-+Removed+deform.js%3AfocusFirstInput().+Forms+have+an+optional+focus+parameter%0A++(%27on%27%7C%27off%27)%3A%0A%0A++++-+%27on%27%3A+if+the+form+contains+an+input+with+autofocus%3D%27on%27%2C+that+input%0A++++++field+will+receive+focus.%0A++++-+%27on%27%3A+if+the+form+does+not+contain+an+input+with+autofocus%3D%27on%27%2C+the%0A++++++first+input+field+of+the+form+will+receive+focus.%0A++++-+%27off%27+disables+focusing+for+that+form.%0A%0A++Default+behaviour+is+%27on%27+(unchanged).%0A&from=rst&to=html5